diff --git a/docs/buildsystem-and-repos.md b/docs/buildsystem-and-repos.md
index 2f49015..498e560 100644
--- a/docs/buildsystem-and-repos.md
+++ b/docs/buildsystem-and-repos.md
@@ -153,7 +153,7 @@ rofiles-fuse exampleos-build mnt
 # Now run global "triggers", generate cache files:
 ldconfig -r mnt
   (Insert other programs here)
-fusermount -u mnt
+fusermount3 -u mnt
 ostree --repo=build-repo commit -b exampleos/x86_64/standard --link-checkout-speedup exampleos-build
 ```
 
diff --git a/man/rofiles-fuse.xml b/man/rofiles-fuse.xml
index c396025..29e0085 100644
--- a/man/rofiles-fuse.xml
+++ b/man/rofiles-fuse.xml
@@ -81,7 +81,7 @@ $ mkdir mnt/anewdir
 $ rm mnt/someoriginalcontent -rf
 
 # Commit and cleanup
-$ fusermount -u mnt
+$ fusermount3 -u mnt
 $ ostree --repo=repo commit --link-checkout-speedup -b somebranch -s 'Commit new content' --tree=dir=branch-checkout
 $ rm mnt branch-checkout -rf
 	</programlisting>
diff --git a/src/rofiles-fuse/main.c b/src/rofiles-fuse/main.c
index abf355e..8ca33fe 100644
--- a/src/rofiles-fuse/main.c
+++ b/src/rofiles-fuse/main.c
@@ -27,7 +27,7 @@
 #include <err.h>
 #include <errno.h>
 #include <fcntl.h>
-#include <fuse.h>
+#include <fuse3/fuse.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
diff --git a/tests/libtest.sh b/tests/libtest.sh
index 24125b2..f775ad2 100755
--- a/tests/libtest.sh
+++ b/tests/libtest.sh
@@ -715,7 +715,7 @@ skip_one_without_strace_fault_injection() {
 }
 
 skip_without_fuse () {
-    fusermount --version >/dev/null 2>&1 || skip "no fusermount"
+    fusermount3 --version >/dev/null 2>&1 || skip "no fusermount3"
 
     capsh --print | grep -q 'Bounding set.*[^a-z]cap_sys_admin' || \
         skip "No cap_sys_admin in bounding set, can't use FUSE"
diff --git a/tests/test-demo-buildsystem.sh b/tests/test-demo-buildsystem.sh
index 2dc78dc..50995b8 100755
--- a/tests/test-demo-buildsystem.sh
+++ b/tests/test-demo-buildsystem.sh
@@ -55,7 +55,7 @@ exampleos_recompose() {
     # Now that we have our rootfs, run triggers
     rofiles-fuse exampleos-build mnt
     demo_triggers mnt/
-    fusermount -u mnt
+    fusermount3 -u mnt
     
     # Then we commit it, using --link-checkout-speedup to effectively
     # only re-checksum the ldconfig file.  We also have dummy commit
diff --git a/tests/test-rofiles-fuse.sh b/tests/test-rofiles-fuse.sh
index b47bb36..c2d7e79 100755
--- a/tests/test-rofiles-fuse.sh
+++ b/tests/test-rofiles-fuse.sh
@@ -37,7 +37,7 @@ $OSTREE checkout -H test2 checkout-test2
 
 rofiles-fuse checkout-test2 mnt
 cleanup_fuse() {
-    fusermount -u ${test_tmpdir}/mnt || true
+    fusermount3 -u ${test_tmpdir}/mnt || true
 }
 libtest_exit_cmds+=(cleanup_fuse)
 assert_file_has_content mnt/firstfile first
@@ -133,7 +133,7 @@ echo "ok flock"
 
 copyup_reset() {
     cd ${test_tmpdir}
-    fusermount -u mnt
+    fusermount3 -u mnt
     rm checkout-test2 -rf
     $OSTREE checkout -H test2 checkout-test2
     rofiles-fuse --copyup checkout-test2 mnt
